PiAUISuite/VoiceCommand/commands.conf

```diff
@@ -1,7 +1,7 @@
 #This is the default config file
 #These are the special options you can set (remove the #)
 #!verify==1
-!keyword==mascot
```

Comment:
hm. It looks like Joice already did your task for you, in that case I'm going to close this PR and just close your issue. I'll try to find another low priority task that's of interest and can teach you something.

Reply:
Oh okay thanks.